src/components/ToolMenu/LayerTree/index.tsx

```diff
@@ -221,6 +232,24 @@ export const LayerTree: React.FC<LayerTreeProps> = ({
               className='layer-name'
             >
               {layer.get('name')}
+              {layer.get('searchable') && shouldDisplayIcon && (
+                < FontAwesomeIcon
+                  icon={faMagnifyingGlass}
+                  className='layer-icon'
+                />
+              )}
+              {layer.get('hoverable') && shouldDisplayIcon && (
+                < FontAwesomeIcon
+                  icon={faCircleInfo}
+                  className='layer-icon'
+                />
+              )}
+              {layer.get('editable') && shouldDisplayIcon && (
+                < FontAwesomeIcon
+                  icon={faPen}
+                  className='layer-icon'
+                />
+              )}
```

Comment:
The current placement will move the loading indicator (see `loading-dots`) to the right and after the newly added icons. Is this intended?

Reply:
This was not intended, i'll remedy that